checker/build.xml

```diff
@@ -777,6 +777,7 @@
         <antcall target="check-compilermsgs"/>
         <antcall target="check-purity"/>
 
+        <antcall target="check-nullness"/>
```

Comment:
This takes a considerable time and generates >1200 warnings. I wanted to add it to make sure there are no exceptions thrown. However, as we already run check-compilermsgs and check-purity, we might not need this.
Two options:
- disable just check-nullness again
- put all three check-xxx targets into a separate travis configuration